test/unit/stopword-highlights.test.js

```diff
@@ -10,6 +10,7 @@ import {
 } from "../../src/stopword-highlights.js";
 
 const genderWords = ["gender", "transgender", "gender-normative", "Gender"];
+const highlightTag = "<strong";
```

Comment:
This could be called `highlightTagMatch` or something similar, since it's more of a pattern that identifies the presence of a tag.

Reply:
Good point. I renamed it `highlightTagPattern` since it's used as a regular expression pattern. I would use the noun "match" to refer to the results found by applying the pattern.